### PR TITLE
doma: Deploy Android build artifacts

### PR DESCRIPTION
**Pull request by andr2000**
1. Deploy build artifacts
2. Provide xl's doma configuration file
3. Rename domu.dts to doma.dts
4. Use Android kernel in Dom0's root filesystem

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend [hunk 1]
```diff
@@ -11,6 +22,27 @@ EXTRA_OEMAKE_append = " TARGET_BOARD_PLATFORM=${SOC_FAMILY}"
 SOC_FAMILY_r8a7795 = "r8a7795"
 SOC_FAMILY_r8a7796 = "r8a7796"
 
+
```
Comment (lorc) [inline recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend]: Why do you need extra line there?

Comment (andr2000) [reply recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend]: my bad

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma.cfg [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+kernel = "/xt/doma/Image"
+
+device_tree = "/xt/domd/doma.dtb"
+
+coproc=[ "/soc/gsx@fd000000" ]
+irqs=[ 151 ]
+iomem=[ "0xfd010,1@0xfd000" ]
+
+# Kernel command line options
+extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/doma ip=192.168.1.12 rw rootwait console=hvc0 cma=256M"
```
Comment (aanisov) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma.cfg:16]: I doubt your Android will boot with this command line.

**Review by aanisov — APPROVED** [review]

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma.cfg (hunk 2 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma.cfg:16]: This is by intention to be able to boot with Android kernel, but prod devel's domu rootfs

**Pull request merged**